sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1442,6 +1446,10 @@ def _mapping_event_from_change_event(self, status, port_dict):
         return event
 
     def task_worker(self, stopping_event, sfp_error_event):
+        global xcvr_table_helper
```

Comment:
lets NOT refer to global instance, so that we have local instance

Reply:
The problem is that `xcvr_table_helper` has been referenced as a global in many places in the `xcvrd` code. Same as the previous one.